.github/workflows/therock-ci.yml

```
@@ -27,17 +41,29 @@ jobs:
     outputs:
       jobs: ${{ steps.jobs.outputs.jobs }}
     steps:
+      # Checkout the mono-repo which includes the PR.
       - name: Checkout rocm-libraries Repository
         uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
         with:
-          repository: "ROCm/rocm-libraries"
+          path:          
+            rocm-libraries
 
       - name: Checkout TheRock Repository
         uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
         with:
           repository: "ROCm/TheRock"
           path: "TheRock"
-          ref: users/geomin12/monorepo-therock-test
+          ref: users/sosa/monorepo-therock-test
+
+      - name: Set up Python
+        uses: actions/setup-python@a26af69be951a213d495a4c3e4e4022e16d87065 # v5.6.0
+        with:
+          python-version: '3.12'
+
+      - name: Patch monorepo
+        run: |
+          python TheRock/build_tools/github_actions/patch_monorepo.py \
```

The location for the script will be
```suggestion
          python TheRock/build_tools/patch_monorepo.py \
```
if we don't move it to `github_actions`.

---

I think since this is specific to github actions, we can keep it there!

---

Let me rephrase: This has never been in a `github_actions` subdirectory and as soon as I merge my PR it will be in the path suggested above.

---

Oh apologies, I was looking at the wrong script!